src/coffee/directives/api/map.coffee

```diff
@@ -36,7 +36,7 @@ angular.module('uiGmapgoogle-maps.directives.api')
         transclude: true
         replace: false
         #priority: 100,
-        template: '<div class="angular-google-map"><div class="angular-google-map-container"></div><div ng-transclude style="display: none"></div></div>'
+        template: '<div class="angular-google-map"><div class="angular-google-map-container" id="angular-google-map-container"></div><div ng-transclude style="display: none"></div></div>'
```

Comment:
There can be more than one map so this id would conflict. If you need get the map via something then find it by css tags and the pick first or second element of those.